test/controllers/git-tab-controller.test.js

```diff
@@ -713,13 +713,15 @@ describe('GitTabController', function() {
         const repository = await buildRepository(workdirPath);
         sinon.spy(repository, 'undoLastCommit');
         fs.writeFileSync(path.join(workdirPath, 'new-file.txt'), 'foo\nbar\nbaz\n');
+        const coAuthorName = 'Janelle Monae';
```

Comment:
one thing I learned is that `fubar` is a military term for "effed up beyond all recognition."  Which uhhh, isn't an image I really want to call to mind, so I'm going to try to use more creative examples as much as I can.

Reply:
Hah, sure, that's fair.

I've been tending to use `aaa / bbb / ccc` style placeholders, or suffixing with a one-up number (`user0`, `user1`). If I can I'll also try to name things after the role I'm intending them to play in the test (`Pull Request With An Empty Description`).

... But I get lazy sometimes too :wink: